tests/fault_injector/test_affected_element_does_not_exist.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 
 from src.base_model import BaseModel
+from src.event_bus.event_bus import EventBus
 from src.fault_injector.fault_configurations.fault_configuration import (
     FaultConfiguration,
 )
```

Comment:
Further down there is still the Logger as an import. Do you still need that?

Reply:
I will remove it.